load.php

```diff
@@ -0,0 +1,17 @@
+<?php
+/**
+ * Altis Core Loader.
+ *
+ * @package altis-core
+ */
+
+namespace Altis\Core;
+
+const ALTIS_CORE_DIR = __DIR__;
+
+include ALTIS_CORE_DIR . '/inc/config.php';
+
+// Define the env type as local if not set higher up the chain.
```

Comment:
I wonder if there's a better way to handle the `HM_ENV_TYPE` default?

Reply:
It's done in the `hm-platform/load.php` file in the beta branch so this is pretty much copy paste from there. I think it should happen at a low level but perhaps in the wp-config.php in the CMS module? Needs to happen before the config is called anywhere.